tools/hpcc_aks_util.pl

```diff
@@ -0,0 +1,772 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long;
+use Pod::Usage;
+
+# Function to show usage
+sub show_usage {
+    print <<EOF;
+Usage: $0 -g RESOURCE_GROUP --name CLUSTER_NAME --namespace NAMESPACE [--show-pods] [--show-nodes] [--ea-discount PERCENT] [--ds_v4_family_discount PERCENT]
+
+Required options:
+  -g, --resource-group RESOURCE_GROUP   Azure resource group name
+  --name, --cluster-name CLUSTER_NAME   AKS cluster name
+  --namespace NAMESPACE                 Kubernetes namespace
+
+Optional options:
+  --show-pods                           Show pods in the output
+  --show-nodes                          Show detailed node information with pods per node
+  --ea-discount PERCENT                 Enterprise Agreement discount percentage (default: 10)
+  --ds_v4_family_discount PERCENT       Dds_v4 family discount percentage for D16ds v4 and larger (default: 10)
```

Comment:
fine for now, but would be good to generalize this to something like
--discountnode "<vmtype>=<pct>"